clap_derive/tests/subcommands.rs

```diff
@@ -185,13 +186,14 @@ fn enum_in_enum_subsubcommand() {
 }
 
 #[test]
+#[ignore]
```

Comment:
Not working, I'm aware, will fix

Reply:
It tests an undocumented feature and I highly suspect that it wasn't intended to work that way, see my comment below.